config.json

```diff
@@ -1632,11 +1681,19 @@
       {
         "groupId": "androidx.security",
         "artifactId": "security-crypto",
-        "version": "1.0.0",
-        "nugetVersion": "1.0.0.19",
+        "version": "1.1.0-alpha06",
```

Comment:
We need to leave these unstable versions of `security-crypto*` off of the `main` branch.